javaslang-benchmark/src/test/java/javaslang/benchmark/JmhRunner.java

```diff
@@ -9,30 +9,38 @@
 import java.util.concurrent.TimeUnit;
 
 public class JmhRunner {
+    public static final int FORKS = 1;
+
     private static final int WARMUP_ITERATIONS = 20;
     private static final int MEASUREMENT_ITERATIONS = 30;
+    public static final int DURATION_MILLIS = 500;
 
     private static final int QUICK_WARMUP_ITERATIONS = 10;
     private static final int QUICK_MEASUREMENT_ITERATIONS = 10;
+    public static final int QUICK_DURATION_MILLIS = 200;
 
     public static void run(Class<?> benchmarkClass) {
-        runAndReport(benchmarkClass, WARMUP_ITERATIONS, MEASUREMENT_ITERATIONS, 500, PrintGc.Enable, Assertions.Disable);
+        runAndReport(benchmarkClass, WARMUP_ITERATIONS, MEASUREMENT_ITERATIONS, DURATION_MILLIS, PrintGc.Enable, Assertions.Disable, FORKS);
+    }
+
+    public static void runDebug(Class<?> benchmarkClass) {
```

Comment:
this is for testing that the benchmarks run correctly (enabling debugging also via `fork = 0`)